modules/virt-about-vm-templates.adoc

```diff
@@ -22,3 +22,4 @@ In the Templates tab, you cannot edit or delete Red Hat Supported or Red Hat Pro
 Using a Red Hat template is convenient because the template is already preconfigured. When you select a Red Hat template to create your own custom template, the *Create Virtual Machine Template* wizard prompts you to add a boot source if a boot source was not added previously. Then, you can either save your custom template or continue to customize it and save it.
 
 You can also select the *Create Virtual Machine Template* wizard directly and create a custom virtual machine template. The wizard prompts you to provide configuration details for the operating system, flavor, workload type, and other settings. You can add a boot source and continue to customize your template and save it.
+If in any case a reference to the templates is needed, they can be found [here](https://github.com/kubevirt/common-templates) and can be used has guidelines to create new templates. 
```

Comment:
Templates sourced from the KubeVirt GitHub repo are not supported by Red Hat. We also usually can't link to GitHub in product docs (as noted in [the contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites)).